### DISCUSSION TITLE
Does this work with steam os 3.5??

### DISCUSSION THREAD

**Discussion by Razpor** [moraroy/NonSteamLaunchers-On-Steam-Deck#132] [General]
Does it still work with steam os 3.5??

**Comment by moraroy** [comment]
@Razpor i have no idea i dont have 3.5 yet

**Reply by MrArdillo** [reply]
It doesn't work on 3.5 preview at the moment

**Reply by moraroy** [reply]
@MrArdillo do me a favor, go into the discovery store and download and run proton Up-Qt or whatever it's called, install the latest GE-Proton-8-22 from there instead. And then run NonSteamLaunchers. According to your logs it wasn't installing because it was missing the proton file.

**Reply by MrArdillo** [reply]
Yes, now worked. Thanks!

**Reply by moraroy** [reply]
@MrArdillo so this in fact does work with 3.5 then?

**Reply by MrArdillo** [reply]
Yes, after installing that version of proton  I installed uplay, ea app, ps plus and battlenet without any issues. Can the installer in the future detect if the version is installed and if is not installed install it before do everything?

**Reply by moraroy** [reply]
The installer already does do that, the problem is for some reason it's not extracting it properly... I still need to figure out why. @MrArdillo

**Reply by MrArdillo** [reply]
Aw okay, i remember seeing it rapidly passing by in the instalation process yes
